app/src/main/java/com/mathroule/demoapp/Helper.java

```diff
@@ -0,0 +1,12 @@
+package com.mathroule.demoapp;
+
+import java.util.Random;
+
+public class Helper {
+
+    public static String PASSWORD = "password";
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/e98ef9725ea3c347435a8b4a618b268d3b79e189/app/src/main/java/com/mathroule/demoapp/Helper.java#L7"></span>com.mathroule.demoapp.Helper.PASSWORD isn’t final but should be</td>
    </tr>
  </tbody>
</table>